### ISSUE TITLE
False positive for 'variable is mutated but never accesed'

### ISSUE THREAD

**Issue by Gethe** [mpeterv/luacheck#72]
If you assign a table while using either `or` or `and`, it will mark it with this warning.
``` lua
local table = _G.other or {1,2,3}
table.key = "value"
```

Removing either side of the operator will clear the warning.

**Comment by mpeterv** [comment]
Thanks for the report. I've just fixed it on master branch, let me know if it works. I'll make a bugfix release later today.

**Comment by Gethe** [comment]
Yup, that works 👍  Thanks for the prompt fix.
